buildozer/targets/osx.py

```diff
@@ -94,6 +94,8 @@ def build_package(self):
 
         self.logger.info('Create {}.app'.format(package_name))
         cwd = join(self.buildozer.platform_dir, 'kivy-sdk-packager-master', 'osx')
+        osx_deps = open(join(cwd, 'requirements.txt')).read()
+        app_deps += osx_deps
         # remove kivy from app_deps
         app_deps = [a for a in app_deps.split('\n') if not a.startswith('#') and a not in ['kivy', '']]
```

Comment:
Question: requirements.txt files can contain quote complex entries, specifying versions based on platform, Python versions etc.

Are these parsed by app_deps?

Reply:
We better ask @akshayaurora (code is from 5 years back in time). But as you see parsing is quite simple...